hera_cal/abscal_funcs.py

```diff
@@ -614,7 +614,8 @@ def merge_gains(gains):
 
 def apply_gains(data, gains, gain_convention='divide'):
     """
-    apply gain solutions to data
+    Apply gain solutions to data. If a requested antenna doesn't
```

Comment:
Can we just eliminate this function?

Reply:
i think we should keep it around for the time being, seeing as its a little different than `recalibrate_in_place`, it may have a use-case. Perhaps at some point after we get IDR2.1 finished we can do a cleaning-of-house for the whole repo?

Reply:
Ok by me.